core/src/com/unciv/logic/battle/Battle.kt

```diff
@@ -578,14 +578,19 @@ object Battle {
     }
 
     private fun tryInterceptAirAttack(attacker: MapUnitCombatant, defender: ICombatant) {
+        if (attacker.unit.hasUnique("Cannot be intercepted")) return
+        // Deprecated in 3.15.6
         if (attacker.unit.hasUnique("Can not be intercepted")) return
```

Comment:
I personally like to indent the deprecated part, so it is clear what is deprecated and what isn't, but this is of course personal choice
```
        // Deprecated in 3.15.6
            if (attacker.unit.hasUnique("Can not be intercepted")) return
        //
```

Reply:
That does indeed improve clarity! I have changed this.